modules/server/src/topics/actionkeybindtopic.cpp

```diff
@@ -75,33 +88,13 @@ std::vector<nlohmann::json> ShortcutTopic::shortcutsJson() const {
             // only one of which is actually defined
             continue;
         }
-
-        const KeyWithModifier& k = keyBinding.first;
-        // @TODO (abock, 2021-08-05) Probably this should be rewritten to better account
-        // for the new action mechanism
-        const interaction::Action& action = global::actionManager->action(
-            keyBinding.second
-        );
-
-        const nlohmann::json shortcutJson = {
-            { "key", ghoul::to_string(k.key) },
-            { "modifiers",
-                {
-                    { "shift" , hasKeyModifier(k.modifier, KeyModifier::Shift) },
-                    { "control" , hasKeyModifier(k.modifier, KeyModifier::Control) },
-                    { "alt" , hasKeyModifier(k.modifier, KeyModifier::Alt) },
-                    { "super" , hasKeyModifier(k.modifier, KeyModifier::Super) }
-                }
-            },
-            { "action", action.identifier },
-        };
-        json.push_back(shortcutJson);
+        nlohmann::json keybindJson = jsonKeybind(keyBinding.first, keyBinding.second);
```

Comment:
Do we want to keep Alex's TODO-comment here? 

(I'm personally not sure. It's not completely obvious what he means with it, but I guess it's the pair representation that could be rewritten?)